README.md

```diff
@@ -108,9 +120,7 @@ Write this grammar using EBNF with common extensions
 
 
 ## A2.3
-
-«replace this with your answer»
-
+![](FSM.png)
```

Comment:
# 5

This is correct, but contains an unnecessary state. You could remove S2 and have S1 loop back to itself on receiving an adjective.